pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -67,6 +69,10 @@ func NewManager(
 	recorder record.EventRecorder,
 	nodeRef *api.ObjectReference,
 	clock util.Clock) (Manager, lifecycle.PodAdmitHandler, error) {
+
+	// TODO: pass this into the manager
+	withImageFs := false
```

Comment:
Could you put with ImageFs inside the managerImpl struct? Then in the subsequent PR you can pass in the boolean to NewManager() to set it up properly.